### ISSUE TITLE
List Export

### ISSUE THREAD

**Issue by ERwoodside** [danniehansen/activity-timer#89]
With data exporter - time tracker, is there a way to include the list name/board name. Now it is part of the filtered option but not exportable. We are thinking to expand to multiple boards and need to confirm this will be an option soon?

Thank you, 
Jodi

**Comment by danniehansen** [comment]
@ERwoodside hi,

That's an excellent idea. I will add this to the export this weekend 👍

**Comment by ERwoodside** [comment]
Dannie,

Would you be open to discussing a few other enhancements (even for
purchase):

   - export all workbooks in a workspace at one time
   - ability to filter all boards by workspace (so user can see full daily
   time recorded)


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by danniehansen** [comment]
@ERwoodside I'm currently booked up with work. So i won't be able to dedicate much time away from that. Could you put some extra words on your wishes? Then i can see if they fit my vision for ActivityTimer and add them in a different release.

Had to postpone the addition of your past suggestions. Something came up this weekend. Will look at it during this week and get it added.

**Comment by danniehansen** [comment]
@ERwoodside I've looked into your request for adding board/list name to the export. I found that list id and name was already available in the pro version. To further accommodate your request, I've added board id/name. These settings are however only available in the pro version.

**Comment by danniehansen** [comment]
@ERwoodside closing the issue for now. Please just comment to re-open.

**Comment by ERwoodside** [comment]
Hi Dannie,

How do I purchase the Pro version, there was only one option...thanks!


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by ERwoodside** [comment]
Hi Dannie,

I have the pro version, this is what I see, that Lists I can filter by, but
the export doesn't include board title or list ID/name.  Thanks!

[image: image.png]


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by ERwoodside** [comment]
Furthermore, is there a way to stop the timer when the browser closes?


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by danniehansen** [comment]
@ERwoodside My apologies for not getting back earlier. Not getting notifications on the issue you're replying to because it's closed.

List name/Board name isn't included in the default columns shown. You can to construct your own view by selecting these from the 'Columns' dropdown. In there you'll find additional columns with more information.

Regarding stopping on browser close. Unfortunately, this isn't possible due to the limitations on the tracking being client-side. If Activity timer in the coming future at some point start to bring in more stable revenue, then we're thinking of reworking the client to track server-side. This will have a bunch of things we can benefit from such as that you're describing with disconnects also disrupting the tracking. But for the time being, this isn't possible unfortunately.

**Comment by ERwoodside** [comment]
Thanks Dannie,

In order to export the data with the List name/Board Name is it possible to
add it to the default columns shown (happy to discuss payment for this
addition).


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by danniehansen** [comment]
Hi @ERwoodside,

I do agree that it's tedious having to choose the same columns over and over every time when wanting to export. I usually doesn't do contract work as i try to keep this funded by the expansion of premium users. This way i ensure i don't entangle the project in privately funded features that may cloud my judgement in terms of what is right for the future of this powerup.

Instead i would opt to allow you to define the default exportable columns (if premium) in the powerup settings. How does that sound? Would that match your needs? This would then have to be configured once for each board that you add the powerup to.

**Comment by ERwoodside** [comment]
Dannie,

That would be great! Let me know once that is ready to go.


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by danniehansen** [comment]
@ERwoodside cool. I think I'll have some time this weekend to look into it. I will notify you once out to get some feedback 👍

**Comment by ERwoodside** [comment]
Thanks Dannie


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by ERwoodside** [comment]
HI Dannie,

I wanted to check in on the status of this.

Thank you!


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by danniehansen** [comment]
Hi @ERwoodside,

Sorry for not getting back to you. I've had a busy few weeks. I'm planning to do the feature this Sunday and release it there. I'll instead of making settings for it just have the export modal remember the previous settings you've used. This way you can set it all up once and just re-use the same settings :) Hope that also fits your use-case.

**Comment by danniehansen** [comment]
@ERwoodside Work are coming along. Doing it at the same time as maintenance. Release is planned for Wednesday. Will keep you posted.

**Comment by danniehansen** [comment]
@ERwoodside I've just released a new update that makes it remember the columns configuration under data export. The setting is remembered on your local computer & is only for the 'Columns' configuration at this point.

**Comment by ERwoodside** [comment]
Hi Dannie,

We had an issue where cards over 100 hours, the activity timer stops
working. Most recently we had an issue where an entire user couldn't use
the board (I suspect because they reached a threshold). Can you advise on
card and board time thresholds? Or settings that might prevent a user from
using the board?

Thank you,


Jodi Woodside

Director of Development & Operations | Elite Research, LLC

Project Manager | Divergent Web Solutions, LLC

9901 Valley Ranch Pkwy. E., Ste. 2035, Irving, TX 75063

Phone: 972-538-1374 | 1-800-806-5661 ext. 714

www.eliteresearch.com | www.divergentwebsolutions.com

Fax: 1-800-806-5661 | ***@***.***

LinkedIn: https://www.linkedin.com/in/jodiwoodside/


Note:  All e-mails and files transmitted from Elite Research, LLC may
contain PRIVILEGED or CONFIDENTIAL information and may be used only by the
intended recipient.  Any use - dissemination, distribution, forwarding,
printing or copying of e-mails or any files (including templates) is
strictly prohibited.

**Comment by danniehansen** [comment]
Hi @ERwoodside,

Please open a new issue if there is anything else unrelated to the current issue.

Trello had an incident not too long ago that causes world-wide disruption for most powerups.

Reported: https://community.developer.atlassian.com/t/csp-unsafe-eval-since-last-evening/66818

This incident can cause issues with powerups not loading & was fixed not long after. However, many clients had the old version cached from Trello and that meant it continues to be an issue even after they fixed it in their end until clients cache invalidates or if they clear it manually.

My suggestion is to ask the user to clear their browser cache and attempt again.

Now, around the issue with 100+ hours. The powerup uses the built-in data storage that Trello allows on each card. However, this storage has limitations in size. If you're having 1 card with too many trackings, then at some point it will fill up the entirety of this allowed space and then stop working. My suggestion here is to not have more than ~50/75 tracking on each card. When you exceed this, then to create a new card for continued tracking.

I do understand that this limitation can be annoying and cause issues. I'm considering adding a built-in measurement of the space to alert the user ahead of time before it get's to a point where it stops working. Such that you can at that point create a new card.

Added a new issue for this over at: https://github.com/danniehansen/activity-timer/issues/98

Lets continue the discussion on this there.
